bin/legacypipe-env

```diff
@@ -0,0 +1,39 @@
+## bashrc snippet for setting up environment for runbrick.py
+
+unset PYTHONPATH
+
+export NERSC_HOST=`/usr/common/usg/bin/nersc_host`
+
+# Where's the code?
+# export LEGACY_PIPE_DIR=/global/cscratch1/sd/desiproc/${NERSC_HOST}-code/legacypipe/
+
+# Where are inputs and outputs going?
+export LEGACY_SURVEY_DIR=/global/cscratch1/sd/desiproc/dr7
+
+export DUST_DIR=/global/cscratch1/sd/desiproc/dust/v0_0
+
```

Comment:
Didn't we say we would use the dust maps in /project?  This will ensure the maps survive scratch-purging from DR to DR.